apps/air-discount-scheme/web/screens/Admin/Admin.tsx

```diff
@@ -92,6 +93,48 @@ const Admin: Screen = ({}) => {
     setFilters(data)
   }
 
+  const downloadCSV = () => {
+    Object.keys(Airlines)
+      .filter(
+        (airline) => !filters.airline || filters.airline?.value === airline,
```

Comment:
This has to be changed due to the new cooperation field. See the `getFlightLegs` function in `./components/Summary/Summary.tsx` + the `airline` object.

They should be moved to `./utils.ts` as well and reused.